Reference/Security/sensitive-data.md

```diff
@@ -5,20 +5,51 @@ versionTo: 10.0.0
 
 # Sensitive data
 
-Marking fields as sensitive will hide the data in those fields for backoffice users that have no business viewing personal data of members.
+Marking fields and properties on member data as sensitive will hide the data in those fields for backoffice users that are not privy to the data.
 
-In order to start marking fields as sensitive, you can add the users who need access to this data to the "Sensitive data" user group. In the User management sections go to Groups and choose the Sensitive data group.
+In this article you will get an overview on how you can grant and/or deny your users access to sensitive data as well as how to mark data as sensitive.
+
+## Grant or deny access to sensitive data
+
+Every new Umbraco installation ships with a default set of User Groups. One them is the **Sensitive data User Group**. In order to give users in the backoffice access to view and work with sensitive data, they need to be part of the Sensitive data User Group.
+
+Any users who are not part of the Sensitive data User Group, will not be able to see the data in the properties that are marked as sensitive. Instead they will will see a generic message: "*This value is hidden. If you need access to view this value please contact your website administrator.*"
+
+![Sensitive data hidden](images/sensitive-data-hidden-v8.png)
+
+While not part of the Sensitive data User Group it is also not possible to export members or member data.
+
+Follow these steps in order to grant a user access to sensitive data.
+
+* Navigate to the **User** section in the Umbraco backoffice.
+* Select the **Groups** menu in the top-right corner.
```

Comment:
To access the Groups menu, we have to click on users in the left pane and then it shows up. 
![image](https://user-images.githubusercontent.com/82437098/175531302-d84383c8-3016-496e-a832-a2008e0e0e43.png)

Maybe add or update this step?

Reply:
Good point. I believe that step is only a thing when you have Umbraco Forms installed though 🤔 
I suggested to add another step.